### PR TITLE
V3 deployment page work

### PR DESCRIPTION
**Pull request by tombee**

### CODE REVIEW

File: controller/static/bower.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "angular-resource": "~1.3.15",
     "angular-jwt": "~0.0.6",
     "jquery-tablesort": "*",
-    "angular-sanitize": "~1.3.15"
+    "angular-sanitize": "~1.3.15",
+    "angular-semantic-ui": "~0.0.2"
```
Comment (ehazlett) [inline controller/static/bower.json:25]: Why the switch to using this?  It seems less stable.

Comment (tombee) [inline controller/static/bower.json:25]: We're only using a simple dropdown directive from it -- we'd probably end up implementing a dropdown directive ourselves anyway eventually to clean up the dropdown usage (similar to what we had in V2).

Comment (tombee) [inline controller/static/bower.json:25]: Plus.. the semantic-ui dropdown javascript doesn't work properly when dynamically adding more `select` (that are using `ngModel`) fields to the view.

Comment (ehazlett) [inline controller/static/bower.json:25]: ok cool.  thanks!

**Comment by ehazlett** [comment]
LGTM

**Pull request merged**